labgrid/remote/client.py

```diff
@@ -1625,6 +1636,13 @@ def main():
                            help="release a place even if it is acquired by a different user")
     subparser.set_defaults(func=ClientSession.release)
 
+    subparser = subparsers.add_parser('release-from',
```

Comment:
I am not in favor of adding another top level command, I think adding a `--from` parameter to the release command is clearer and does not clutter the interface as much.

Reply:
As above, the use case and behavior (especially error reporting) is different, so it's find it confusing to have it as a flag to relase.